I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 455, in script
    l "Тест це не 100% гарантії."
ValueError: unsupported format character '?' (0x433) at index 16

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 455, in script
    l "Тест це не 100% гарантії."
  File "D:\Programs\renpy-8.0.0-sdk\renpy\ast.py", line 901, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "D:\Programs\renpy-8.0.0-sdk\renpy\exports.py", line 1362, in say
    what = what % tag_quoting_dict
ValueError: unsupported format character '?' (0x433) at index 16

Windows-10-10.0.19044 AMD64
Ren'Py 8.0.0.22062402
schi 1.0
Tue Feb  7 02:46:49 2023
